ui/app/pages/send/send.scss

```diff
@@ -14,10 +14,10 @@
     }
 
     .page-container__header-close-text {
+      @include H5;
```

Comment:
Increasing the `line-height` here seems fine. It does push the 'Cancel" button in the send flow down a little bit, but it looks fine and it doesn't move anything else.